lib/Raven/ClientBuilder.php

```diff
@@ -87,6 +90,31 @@ class ClientBuilder implements ClientBuilderInterface
     protected $configuration;
 
     /**
+     * @var UriFactory The PSR-7 URI factory
+     */
+    protected $uriFactory;
```

Comment:
Does it makes sense in a builder class to have the properties as `private`? What if an user needs to customize how the client is created? If we're going to make all properties `private` we could also mark this class as `final` because we don't expose any other public method not defined in the interface